pkg/provider/aws/parameterstore/parameterstore_test.go

```diff
@@ -569,6 +571,17 @@ func TestGetSecret(t *testing.T) {
 		pstc.expectedSecret = "RRRRR"
 	}
 
+	// good case: key is passed in and prefix is set, output is sent back
```

Comment:
The other tests don't really care about the prefix, because everything is a canned response and don't really assert mocks. This makes it super difficult to unit test this feature from PushSecret and DeleteSecret's side. I'll try to do something about it without disrupting everything too much. Maybe I just copy a whole new test case for it. ¯\_(ツ)_/¯

Reply:
Done.